v2/support/secure_settings_sample_app/README.md

```diff
@@ -0,0 +1,36 @@
+# Bag End Catering Helper
```

Comment:
😅 I think the humour here may be a bit distracting and could cause some confusion. Maybe we should prefer a more plain readme for consistency with the other demo apps?

Reply:
Nonsense, this is a perfectly serious Readme! What are you talking about ?!

I might tidy it up just a little, though.